schematypes_test.go

```diff
@@ -42,6 +43,17 @@ func makeErrorResponse() *ErrorResponse {
 	}
 }
 
+func toRawMessage(m interface{}) json.RawMessage {
```

Comment:
Wondering if it is worth exporting these for clients that do not want to define their own types and/or prefer to use maps in tests.

Reply:
Maybe? But the best time for adding new function/methods to the API set is when it is needed - so I am inclined to wait for the actual use case - then examine this is a suitable API and add it when needed.